client/app/layouts/splash.html

```diff
@@ -1,4 +1,18 @@
-<language-chooser></language-chooser>
+<div>
+  <language-chooser></language-chooser>
+  <md-button href="https://github.com/yunity/foodsaving-frontend">
+    <i class="fa fa-github"></i>
+    Github
+    <md-tooltip md-direction="bottom">
```

Comment:
use the translate attribute instead of filters (less watches, less slowdown)
`translate="GLOBAL.GITHUB_NOTE"`